activemq-stomp/src/main/java/org/apache/activemq/transport/stomp/ProtocolConverter.java

```diff
@@ -558,10 +568,6 @@ protected void onStompSubscribe(StompFrame command) throws ProtocolException {
             throw new ProtocolException("SUBSCRIBE received without a subscription id!");
         }
 
-        if (destination == null || "".equals(destination)) {
-            throw new ProtocolException("Invalid empty or 'null' Destination header");
-        }
-
         final ActiveMQDestination actualDest = translator.convertDestination(this, destination, true);
```

Comment:
Why don't we have the ProtocolException if destination is not defined ?

Reply:
Corrected